vocode/streaming/models/synthesizer.py

```diff
@@ -230,18 +230,18 @@ class PollySynthesizerConfig(SynthesizerConfig, type=SynthesizerType.POLLY.value
 
 
 DEFAULT_CARTESIA_MODEL_ID = "sonic-english"
-DEFAULT_CARTESIA_VOICE_ID = "829ccd10-f8b3-43cd-b8a0-4aeaa81f3b30"
+DEFAULT_CARTESIA_VOICE_ID = "f9836c6e-a0bd-460e-9d3c-f7299fa60f94"
 
 
-class CartesiaVoiceControls(TypedDict):
+class CartesiaVoiceControls(BaseModel):
     """See https://docs.cartesia.ai/user-guides/voice-control"""
 
-    speed: str
-    emotion: List[str]
+    speed: Union[str, float] = ""
```

Comment:
@ajar98 Pydantic seems to cast `floats` to `str` while converting to a dict, any recs on fixing this?

Reply:
fixed